pkg/istio-agent/xds_proxy.go

```diff
@@ -586,6 +595,10 @@ func forwardToEnvoy(con *ProxyConnection, resp *discovery.DiscoveryResponse) {
 		proxyLog.Errorf("Skipping forwarding type url %s to Envoy as is not a valid Envoy type", resp.TypeUrl)
 		return
 	}
+	if con.isClosed() {
+		// TODO: do we need to redirect to the new connection
```

Comment:
No IMO, the stream is stateful so we cannot cross connections

Reply:
I was thinking is envot stateful? Will envoy re-request if one req or response is dropped because stream closed

Reply:
We can further discuss in a separate issue maybe